src/figuresClasses.ts

```diff
@@ -1,11 +1,95 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('All sides must be greater then 0');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error(`Sides ${a}, ${b} and ${c} can't form a triangle`);
+    }
+
+    this.color = color;
+
+    this.a = a;
+
+    this.b = b;
+
+    this.c = c;
+  }
```

Comment:
The constructor should not manually assign values to the class properties `color`, `a`, `b`, and `c` since TypeScript can automatically assign parameter values to class properties when using access modifiers like `public` or `private` in the parameter list.